src/Component-Routes/MakeNote/MakeNote.jsx

```diff
@@ -0,0 +1,66 @@
+import './MakeNote.css';
+import { useState } from 'react';
+import { addDoc, collection } from '../../firebase-store/firebase-imports';
+import { auth,db } from '../../firebase-store/firebaseKeys';
+import { useNavigate } from 'react-router-dom';
+import Header from '../Layout-component/Header';
+
+function MakeNote({handleExit, UserInfo}) {
+
+const [ title, setTitle ] = useState('')
+const [ note, setNote ] = useState('')
+const navigate = useNavigate();
+
+// const notesCollectionRef = collection(db, "notes");
+
+const createNote = async () => {
+  await addDoc(collection(db, 'users', auth.currentUser.uid, 'notes'), {
+  title,
+  note,
+  author: auth.currentUser.displayName,
+  id: auth.currentUser.uid,
+})
+navigate('/')
+};
+
+
+const handleChangeTitle = (e)=> {
+setTitle(e.target.value)
+}
+const handleChangeNote = (e)=> {
+setNote(e.target.value)
+}
```

Comment:
Lo mismo con esto, esta raro
```suggestion
const handleChangeTitle = (e)=> {
setTitle(e.target.value)
}
const handleChangeNote = (e)=> {
setNote(e.target.value)
}
```